docs/rest/DaVinciPlanNet/DaVinci_PlanNet.http

```diff
@@ -30,46 +30,359 @@ grant_type=client_credentials
 
 ### Post Insurance Plan Coverage Area Search Parameter Definition 
-POST {{fhirurl}}/SearchParameter
+POST {{fhirurl}}/SearchParameter  HTTP/1.1
 Authorization: Bearer {{token}}
 content-type: {{contentType}}
 
-< ./SearchParameters/InsurancePlanCoverageArea.json
+{
+    "resourceType": "SearchParameter",
+    "id": "insuranceplan-coverage-area",
+    "text": {
+      "status": "generated",
+      "div": "<div xmlns=\"http://www.w3.org/1999/xhtml\"><p><b>Generated Narrative</b></p><p><b>url</b>: <code>http://hl7.org/fhir/us/davinci-pdex-plan-net/SearchParameter/insuranceplan-coverage-area</code></p><p><b>version</b>: 1.0.0</p><p><b>name</b>: Plannet_sp_insuranceplan_coverage_area</p><p><b>status</b>: active</p><p><b>date</b>: May 23, 2018 12:00:00 AM</p><p><b>publisher</b>: HL7 Financial Management Working Group</p><p><b>contact</b>: HL7 Financial Management Working Group: <a href=\"http://www.hl7.org/Special/committees/fm/index.cfm\">http://www.hl7.org/Special/committees/fm/index.cfm</a>,<a href=\"mailto:fm@lists.HL7.org\">fm@lists.HL7.org</a></p><p><b>description</b>: Select products that are offered in the specified location</p><p><b>jurisdiction</b>: <span title=\"Codes: {urn:iso:std:iso:3166 US}\">United States of America</span></p><p><b>code</b>: coverage-area</p><p><b>base</b>: InsurancePlan</p><p><b>type</b>: reference</p><p><b>expression</b>: InsurancePlan.coverageArea</p><p><b>target</b>: Location</p><p><b>multipleOr</b>: true</p><p><b>multipleAnd</b>: true</p></div>"
+    },
+    "url": "http://hl7.org/fhir/us/davinci-pdex-plan-net/SearchParameter/insuranceplan-coverage-area",
+    "version": "1.0.0",
+    "name": "coverage-area",
+    "status": "active",
+    "date": "2018-05-23T00:00:00+00:00",
+    "publisher": "HL7 Financial Management Working Group",
+    "contact": [
+      {
+        "name": "HL7 Financial Management Working Group",
+        "telecom": [
+          {
+            "system": "url",
+            "value": "http://www.hl7.org/Special/committees/fm/index.cfm"
+          },
+          {
+            "system": "email",
+            "value": "fm@lists.HL7.org"
+          }
+        ]
+      }
+    ],
+    "description": "Select products that are offered in the specified location",
+    "jurisdiction": [
+      {
+        "coding": [
+          {
+            "system": "urn:iso:std:iso:3166",
+            "code": "US"
+          }
+        ]
+      }
+    ],
+    "code": "coverage-area",
+    "base": [
+      "InsurancePlan"
+    ],
+    "type": "reference",
+    "expression": "InsurancePlan.coverageArea",
+    "target": [
+      "Location"
+    ],
+    "multipleOr": true,
+    "multipleAnd": true
+  }
 
 ### Post Insurance Plan Plan Type Search Parameter Definition 
-POST {{fhirurl}}/SearchParameter
+POST {{fhirurl}}/SearchParameter HTTP/1.1
 Authorization: Bearer {{token}}
 content-type: {{contentType}}
 
-< ./SearchParameters/InsurancePlanPlanType.json
```

Comment:
>InsurancePlanPlanType [](http://example.com/codeflow?start=21&length=21)

Is there are reason why you prefer to include files inside file rather than separate files?

Reply:
The search parameters weren't all posting if I used a separate file, unlike the profiles & extensions, so I switched to putting them directly in the file @Ivanidzo4ka

Reply:
Is there are any specific error?
I run it on my machine and I can post them, so it seems like working for me. Any chance you would have time to debug it together?

I would prefer to keep them separated, since it's much smaller http file and it's also easier to track changes.

Reply:
@Ivanidzo4ka  I just retested with a clean DB. All but 1 work if I post as separate files. For Organization Affiliation Network: http://hl7.org/fhir/us/davinci-pdex-plan-net/STU1/SearchParameter-organizationaffiliation-network.html , I got an error that it already existed if I posted as a separate file, but it works if I put the body of the SP directly in the HTTP file

Reply:
Resolved